contracts/test/v0.8/automation/KeeperRegistry2_0.test.ts

```diff
@@ -23,6 +23,27 @@ import { MockArbGasInfo } from '../../../typechain/MockArbGasInfo'
 import { MockOVMGasPriceOracle } from '../../../typechain/MockOVMGasPriceOracle'
 import { UpkeepTranscoder } from '../../../typechain/UpkeepTranscoder'
 
+//////////////////////////////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////////////////////////
+
+/*********************************** REGISTRY v2.0 IS FROZEN ************************************/
+
+// All tests are disabled for this contract, as we expect it to never change in the future.
+// Instead, we test that the bytecode for the contract has not changed.
+
+const BYTECODE = KeeperRegistryFactory.bytecode
+const BYTECODE_CHECKSUM =
+  '0x60660453a335cdcd42b5aa64e58a8c04517e8a8645d2618b51a7552df6e2973b'
+
+describe('KeeperRegistry2_0 - Frozen', () => {
```

Comment:
You could use `describe.only` instead of `.skip` lower down to keep all changes in one place

Reply:
we have a flag that disables use of `describe.only` in CI lol (to prevent people from accidentally committing tests with erroneous `.only`)